src/Account/components/TransactionList.tsx

```diff
@@ -106,13 +106,11 @@ function RemotePublicKeys(props: { publicKeys: string[]; short?: boolean; testne
   if (props.publicKeys.length === 0) {
     return <>-</>
   } else if (props.publicKeys.length === 1) {
-    return (
-      <PublicKey publicKey={props.publicKeys[0]} testnet={props.testnet} variant={props.short ? "short" : "full"} />
-    )
+    return <Address address={props.publicKeys[0]} testnet={props.testnet} variant={props.short ? "short" : "full"} />
   } else {
     return (
       <>
-        <PublicKey publicKey={props.publicKeys[0]} testnet={props.testnet} variant="short" />{" "}
+        <Address address={props.publicKeys[0]} testnet={props.testnet} />{" "}
```

Comment:
Don't we still need to keep the `variant="short"`?

Reply:
I decided to set `variant` to `"short"` by default because we almost always use `"short"` anyways.